tests/conftest.py

```diff
@@ -4,8 +4,36 @@
 import sys
 from os.path import abspath, dirname, join
 
+import pytest
+
 
 # allow having multiple repository checkouts and not needing to remember to rerun
 # 'pip install -e .[dev]' when switching between checkouts and running tests.
 git_repo_path = abspath(join(dirname(dirname(__file__)), "src"))
 sys.path.insert(1, git_repo_path)
+
+# import local modules after fixing up sys.path
+if 1:  # flake be quiet
+    from transformers.utils.logging import (
+        logging_level_str_to_code,
+        logging_levels_as_strings,
+        set_global_logging_level,
+    )
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--loglevel",
```

Comment:
Is `--log_level` better?

Reply:
As I wrote in the OP:

> pytest has its own `--log_level` for its own things, so perhaps a different name would be more suitable for the new option? `--globalloglevel`?

`--libs_log_level`, `--pckgs_log_level`?

Reply:
I've got no idea. Naming is so hard 😢. Let's here from @LysandreJik

Reply:
I think I got it `--set_verbosity_all` to match the function